airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -273,6 +273,8 @@ def type_map(cls, mysql_type):
         d = {
             FIELD_TYPE.INT24: 'INTEGER',
             FIELD_TYPE.TINY: 'INTEGER',
+            FIELD_TYPE.BINARY: 'BINARY',
+            FIELD_TYPE.VARBINARY: 'BINARY',
```

Comment:
Types here are all from `MySQLdb.constants.FIELD_TYPE`, but even if in the latest version of `MySQLdb` (https://github.com/farcepest/MySQLdb1/blob/master/MySQLdb/constants/FIELD_TYPE.py),  `BINARY` and `VARBINARY` are not there yet.

Let me know if I missed anything.

Reply:
@XD-DENG 
:( I didn't know what package Airflow uses.
These types are in my production table. 

They are not new in fact they are ancient from 2011 and earlier
https://dev.mysql.com/doc/refman/5.7/en/binary-varbinary.html

Maybe Airflow should consider working with another package if it doesn't support something that fundamental?

Reply:
@XD-DENG  you referred to the wrong repository.
Airflow uses this one:
https://pypi.org/project/mysqlclient/

Reply:
It's the same: https://github.com/PyMySQL/mysqlclient-python/blob/master/MySQLdb/constants/FIELD_TYPE.py

Reply:
I downloaded the .tar.gz file from https://pypi.org/project/mysqlclient/ to double-check as well. Leads to the same conclusion.

Reply:
@XD-DENG  I'm not sure if this is relevant. In any case I opened a ticket https://github.com/PyMySQL/mysqlclient-python/issues/335

Reply:
@RosterIn , your issue is closed and it seems those types are not supported by the lib.

Reply:
@feng-tao @XD-DENG  it's the other way around. They said it is supported and the file you were referring to has nothing to do with it.